content/en/bare_metal_machine.md

```diff
@@ -16,4 +16,4 @@ Pairing one operating system with one physical computer is the original pattern
 
 By dedicating all compute resources of a computer to a single operating system, you potentially provide the best possible performance to the operating system. If you need to run a workload that must have extremely fast access to hardware resources, bare metal may be the right solution. 
 
-In the context of [cloud native apps](/cloud_native_apps/), we generally think of performance in terms of [scaling](/scalability/) to a large number of concurrent events, which can be handled by [horizontal scaling](/horizontal_scaling/) (adding more machines to your resource pool). But some workloads may require vertical scaling (adding more power to an existing physical machine) and/or an extremely fast physical hardware response in which case bare metal is better suited. Bare metal also allows you to tune the physical hardware and possibly even hardware drivers to help accomplish your task.
+In the context of [cloud native apps](/cloud_native_apps/), we generally think of performance in terms of [scaling](/scalability/) to a large number of concurrent events, which can be handled by [horizontal scaling](/horizontal_scaling/) (adding more machines to your resource pool). But some workloads may require [vertical scaling](/vertical_scaling/) (adding more power to an existing physical machine) and/or an extremely fast physical hardware response in which case bare metal is better suited. Bare metal also allows you to tune the physical hardware and possibly even hardware drivers to help accomplish your task.
```

Comment:
Unfortunately, closing and reopening this PR did not triggered Netlify build. 😥
@dipankardas011 Could you push another commit that changes nothing in actual, like adding empty line in the end of the file?

```suggestion
In the context of [cloud native apps](/cloud_native_apps/), we generally think of performance in terms of [scaling](/scalability/) to a large number of concurrent events, which can be handled by [horizontal scaling](/horizontal_scaling/) (adding more machines to your resource pool). But some workloads may require [vertical scaling](/vertical_scaling/) (adding more power to an existing physical machine) and/or an extremely fast physical hardware response in which case bare metal is better suited. Bare metal also allows you to tune the physical hardware and possibly even hardware drivers to help accomplish your task.

```

Reply:
okay pushing a commit🙂